yt/frontends/swift/fields.py

```diff
@@ -0,0 +1,26 @@
+"""
+SWIFT fields
+
+
+
+
+"""
+from __future__ import absolute_import
+
+#-----------------------------------------------------------------------------
+# Copyright (c) 2018, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+from yt.frontends.sph.fields import \
+    SPHFieldInfo
+
+class SwiftFieldInfo(SPHFieldInfo):
```

Comment:
I kinda think we should separate out SPHFieldInfo into a `GadgetFieldInfo` and a `TipsyFieldInfo`, if only to keep things tidy. If you'd be interested in doing that cleanup it would be very much appreciated.

Reply:
i.e get rid of SPHFieldInfo and just have Tipsy and Gadget?
I'm happy to do this, just unsure exactly what is being asked

Reply:
So `SPHFieldInfo` contains both Tipsy and Gadget field names, mostly for historical reasons. It's probably not the best idea to combine them like that. We should separate them. It might ultimately make sense to keep some parts of `SPHFieldInfo` (e.g. some of the methods might be generically useful) but the field names that are particular to each frontend should show up in that frontend, not in a base class that's shared by multiple frontends.

Reply:
I see yep - I can have a look at that!